reference/resources/card_details.yaml

```
@@ -39,13 +39,5 @@ properties:
     example: US
 
   required_fields:
-    type: array
-    minItems: 0
-    items:
-      type: string
-    description: |-
-      A list of fields that are required to process a transaction for this card.
-    example:
-      - "name"
-      - "address"
-      - "tax_id"
+    allOf:
+      - $ref: "../openapi.yaml#/components/schemas/CardRequiredFields"
```

```suggestion
     $ref: "../openapi.yaml#/components/schemas/CardRequiredFields"
```

No need for the allOf, I think.